pom.xml

```diff
@@ -0,0 +1,163 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>uk.gov.companieshouse</groupId>
+	<artifactId>company-accounts-library</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+	<name>company-accounts-library</name>
+
+	<properties>
+
+		<!-- Encoding -->
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+		<!-- Dependency Versions -->
+		<avro.version>1.8.1</avro.version>
+		<commons-codec-version>1.9</commons-codec-version>
+		<commons-io.version>2.4</commons-io.version>
+		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
+		<junit.version>4.12</junit.version>
+		<maven-compiler-plugin.version>3.5.1</maven-compiler-plugin.version>
+		<snakeyaml.version>1.17</snakeyaml.version>
+		<structured-logging.version>1.2.0-rc1</structured-logging.version>
+
+		<!-- JDK -->
+		<source-level>1.8</source-level>
+		<target-level>1.8</target-level>
+
+		<!-- Repositories -->
+		<artifactoryBaseUrl>http://ch-jenkins-2k8.internal.ch:8081/artifactory</artifactoryBaseUrl>
+		<artifactoryResolveRepo>libs-release</artifactoryResolveRepo>
+		<publishRepo>dev</publishRepo>
+
+		<!-- Sonar -->
+		<sonar-maven-plugin.version>3.4.0.905</sonar-maven-plugin.version>
+		<sonar.host.url>${CODE_ANALYSIS_HOST_URL}</sonar.host.url>
+		<sonar.login>${CODE_ANALYSIS_LOGIN}</sonar.login>
+		<sonar.password>${CODE_ANALYSIS_PASSWORD}</sonar.password>
+
+		<structured-logging.version>1.2.0-rc1</structured-logging.version>
+
+	</properties>
+
+	<dependencies>
+		<!-- Compile -->
+		<dependency>
+			<groupId>commons-codec</groupId>
+			<artifactId>commons-codec</artifactId>
+			<version>${commons-codec-version}</version>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>${commons-io.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.avro</groupId>
+			<artifactId>avro</artifactId>
+			<version>${avro.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.sonarsource.scanner.maven</groupId>
+			<artifactId>sonar-maven-plugin</artifactId>
+			<version>${sonar-maven-plugin.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>${snakeyaml.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>uk.gov.companieshouse</groupId>
+			<artifactId>structured-logging</artifactId>
+			<version>${structured-logging.version}</version>
+		</dependency>
+
+		<!-- Test -->
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>${junit.version}</version>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+
+	<profiles>
+		<profile>
```

Comment:
Can we follow the way we do the `java-session-handler` in terms of the build profiles within the `pom.xml`. The reason for this is that due to the new artifactory that exists within AWS, publishing the snapshots and release jars to the cloud would save us/devOps having to migrated and change this `pom.xml` on future